acl2020_tools/awscognito/cognito_users.py

```diff
@@ -96,7 +97,8 @@ def parse_file(path):
     error_message = ""
     users = []
 
-    _, ext = path.split("/")[-1].split(".")
+    results = path.split("/")[-1].split(".")
+    ext = results[-1]
```

Comment:
sometimes the filename may contain multiple dots

Reply:
Thanks a lot.